recipes/mujs/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "mujs" %}
+{% set version = "1.3.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://mujs.com/downloads/mujs-{{ version }}.tar.gz
+  sha256: c015475880f6a382e706169c94371a7dd6cc22078832f6e0865af8289c2ef42b
+  patches:
+    - make.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make
+  host:
+    - readline
+
+test:
+  commands:
+    - touch test.c; mujs -s test.c
```

This should ensure that the second command only happens if the first one is successful. 
```suggestion
    - touch test.c && mujs -s test.c
```